src/ContractsStore.ts

```diff
@@ -0,0 +1,82 @@
+import { PersistableStore } from './PersistableStore'
+import { providers } from 'ethers'
+import ContractSynchronizer, {
+  ContractSynchronizerSchema,
+} from './helpers/ContractSynchronizer'
+import Network from './models/Network'
+import transformObjectValues from './helpers/transformObjectValues'
+
+export class ContractsStore extends PersistableStore {
+  connectedAccounts: { [account: string]: ContractSynchronizer } = {}
+  currentBlock?: number
+  addressToTokenIds?: Promise<{ [address: string]: string[] } | undefined>
+
+  getLogs: (filter: providers.Filter) => Promise<providers.Log[]>
```

Comment:
Can we import this stuff like this:
```TS
import { Log, Filter, etc } from '@ethersproject/providers'
```
?

Reply:
yes, I'll fix that